quint/src/flattening.ts

```diff
@@ -241,6 +241,10 @@ class Flatenner {
     const qualifiedName = def.defName ? undefined : def.qualifiedName ?? def.protoName
 
     const protoModule = this.importedModules.get(def.protoName)!
+    if (!protoModule) {
+      // Something got went really wrong
+      console.error(`Imported module ${def.protoName} not found. Definitions out of order?`)
```

Comment:
I thought this PR adds support for out-of-order defs?

Reply:
It's a guard rail. I have added it before toposort was working, as it was very hard to understand what was going on.

Reply:
It totally does. I have added an explanatory exception there, just in case the parse phase 4 is missing somewhere.